src/Symfony/Component/Serializer/CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 CHANGELOG
 =========
 
+6.2
+---
+
+ * Add `SerializedPath` to flatten nested attributes
+ * `AttributeMetadataInterface` has additional methods: `setSerializedPath()` and `getSerializedPath()`
```

Comment:
Add methods `setSerializedPath()` and `getSerializedPath()` to `AttributeMetadataInterface`